build.gradle

```diff
@@ -47,6 +47,8 @@ apply plugin: 'se.patrikerdes.use-latest-versions'
 def versions = [
         pact_version: '4.1.7'
 ]
+ext['spring-security.version'] = '5.4.5'
```

Comment:
Minor: If we're defining this variable, we might as well use a variable for all of the Spring Security dependencies rather than hardcoding `5.4.5` everywhere?